src/components/Chat/InputMessageV2.tsx

```diff
@@ -194,24 +386,44 @@ const InputMessageV2 = ({
             labelLeft={!inputFocus ? 'Anonymity' : undefined}
             styleLabelLeft={[
               styles.labelToggle,
-              {color: type === 'ANONYMOUS' ? colors.gray : colors.blue}
+              {color: type === ANONYMOUS ? colors.gray : colors.blue}
             ]}
-            onValueChange={toggleChnage}
-            value={type === 'ANONYMOUS'}
+            onValueChange={toggleChange}
+            value={type === ANONYMOUS}
           />
         )}
       </View>
-      <TouchableOpacity
-        style={[styles.btn, isDisableButton() ? styles.disableButton : styles.enableButton]}
-        disabled={isDisableButton()}
-        onPress={handleSendMessage}>
-        <IconSend
-          style={styles.icSendButton}
-          fillBackground={sendButtonStyle()}
-          fillIcon={colors.white}
-        />
-      </TouchableOpacity>
-      <SheetEmoji ref={refEmoji} selectEmoji={onSelectEmoji} />
+      {text?.trim() !== '' && (
+        <TouchableOpacity
+          style={[styles.btn, isDisableButton() ? styles.disableButton : styles.enableButton]}
+          disabled={isDisableButton()}
+          onPress={handleSendMessage}>
+          <IconSend
+            style={styles.icSendButton}
+            fillBackground={sendButtonStyle()}
+            fillIcon={colors.white}
+          />
+        </TouchableOpacity>
+      )}
+      {text?.trim() === '' && (
+        <TouchableOpacity style={styles.btn} onPress={onSelectAttachment}>
+          <IconPlusAttachment style={styles.icSendButton} fillIcon={plusButtonStyle()} />
+        </TouchableOpacity>
+      )}
+
+      <BottomSheetAttachment
+        ref={refAttachment}
+        onOpenMedia={onOpenMedia}
+        onOpenGIF={onOpenGIF}
+        onOpenCamera={onOpenCamera}
+        onOpenFile={onOpenFile}
+        isLoadingUploadMedia={isLoadingUploadImageMedia}
+        isLoadingUploadGIF={false}
+        isLoadingUploadCamera={isLoadingUploadImageCamera}
+        isLoadingUploadFile={false}
+      />
+
+      <BottomSheetGif ref={refGif} onCancel={onCloseGIF} onSelect={handleSelectGIF} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces two new components, `BottomSheetAttachment` and `BottomSheetGif`, which are used for handling attachments in chat messages. These components are passed several props related to handling attachments, including functions for opening media, GIFs, the camera, and files, as well as loading states for each of these actions. This is a good practice as it separates concerns and makes the code more modular.


<!-- This is an auto-generated comment by OSS CodeRabbit -->